docs/node-operators/getting-started.mdx

```diff
@@ -25,54 +25,61 @@ This documentation is for the **Mainnet** release.
 
 ## Requirements
 
-**Software**: macOS, Linux (currently supports Debian 9 and Ubuntu 18.04 LTS), or any host with Docker
+**Software**: Supported environments include macOS, Linux (Debian 10, 11, 12 and Ubuntu 20.04 LTS), and any host machine with Docker. 
```

Comment:
Remove time-bound content to create [timeless documentation](https://developers.google.com/style/timeless-documentation)

basically, no "future" or "currently" or "now"